pxr/usdImaging/lib/usdviewq/appController.py

```diff
@@ -2267,13 +2267,19 @@ def _cleanAndClose(self):
             t.PrintTime('tear down the UI')
 
     def _openFile(self):
-        (filename, _) = QtWidgets.QFileDialog.getOpenFileName(self._mainWindow, "Select file",".")
-        if len(filename) > 0:
+        extensions = Sdf.FileFormat.FindAllFileFormatExtensions()
+        fileFilter = "USD Compatible Files (" + " ".join("*." + e for e in extensions) + ")" 
```

Comment:
@superfunc - my cincern here is that if the list of supported types is long, the .usd* native types might get lost at the end.  Can you flatten the set to a list and pull the usd types out to the front?

Reply:
Yeah good note.